app/server/resolvers/test/index.ts

```
@@ -64,6 +67,15 @@ export const createTestResolver = async (
     { db, logger }
   );
 
+  if (branch && team.git_sync_integration_id) {
```

This should throw an error if !team.git_sync_integration_id